Common/UI/Root.cpp

```diff
@@ -249,6 +258,7 @@ KeyEventResult UnsyncKeyEvent(const KeyInput &key, ViewGroup *root) {
 		retval = KeyEventResult::PASS_THROUGH;
 		break;
 	default:
+		retval = KeyEventResult::ACCEPT;
```

Comment:
What's the purpose of `KeyEventToFocusMoves()`'s logic then?  For example, `return KeyEventResult::IGNORE_KEY;` would just be ignored, so multiple key repeat systems would indeed overlap, right?

-[Unknown]

Reply:
That's a good question. I think the answer is that we should check for repeats again here..